libraries/hotstuff/chain_pacemaker.cpp

```diff
@@ -101,9 +101,12 @@ namespace eosio { namespace hotstuff {
 #endif
 //===============================================================================================
 
-   chain_pacemaker::chain_pacemaker(controller* chain, std::set<account_name> my_producers, fc::logger& logger)
+   chain_pacemaker::chain_pacemaker(controller* chain,
+                                    std::set<account_name> my_producers,
+                                    std::map<fc::crypto::blslib::bls_public_key, fc::crypto::blslib::bls_private_key> finalizer_keys,
```

Comment:
Would be nice to have a type alias defined for ` std::map<fc::crypto::blslib::bls_public_key, fc::crypto::blslib::bls_private_key>`